sycl/test/kernel_param/attr.cpp

```diff
@@ -0,0 +1,22 @@
+// RUN: %clangxx -fsycl-device-only -Xclang -fsycl-is-device -emit-llvm %s -S -o %t.ll -I %sycl_include -Wno-sycl-strict -Xclang -verify-ignore-unexpected=note,warning -Xclang -disable-llvm-passes -D__ENABLE_USM_ADDR_SPACE__
+// RUN: FileCheck %s --input-file %t.ll
+
+// Check copying of parallel_for kernel attributes to wrapper kernel.
+
+#include <CL/sycl.hpp>
+using namespace cl::sycl;
+
+int main() {
+  range<1> Size{10};
+  {
+    queue myQueue;
+    myQueue.submit([&](handler &cgh) {
+      cgh.parallel_for<class C>(Size, [=](item<1> ITEM)
+                                          [[intel::reqd_work_group_size(4)]]{});
+    });
+  }
+
+  return 0;
+}
+
+// CHECK: define {{.*}}spir_kernel void @{{.*}}__pf_kernel_wrapper{{.*}}reqd_work_group_size
```

Comment:
Please add a newline to the end of the file.

Reply:
Ok